### PR TITLE
feat(update-api-changes): ZEUS-2203

### PR DESCRIPTION
**Pull request by stephenhi**
Updated for removal of 'label' as not utilised and added more
detail/examples for the update of metadata attributes endpoint.

### CODE REVIEW

File: reference/Upload-API.yaml [hunk 1]
```diff
@@ -319,65 +330,89 @@ paths:
...
+              Date Value Example:
+                value:
+                  attributes:
+                    - id: 7ac5484e-ec8c-45f6-9437-95c909f5d41d
+                      value: '2022-02-24'
+              Time Value Example:
+                value:
+                  attributes:
+                    - id: 17cfb9e1-1775-41a8-8628-1cf053551c73
+                      value: '15:30'
```
Comment (yordankas) [inline reference/Upload-API.yaml:396]: I tried with 7:14 for example but understand that format could be HH:mm :)

**Review by yordankas — APPROVED** [review]

File: reference/Upload-API.yaml [hunk 2]
```diff
@@ -319,65 +330,89 @@ paths:
                   type: array
                   uniqueItems: true
                   minItems: 1
-                  description: The metadata attributes
+                  description: The metadata attributes (either as a single value or a collectioj of values).
```
Comment (rodek) [inline reference/Upload-API.yaml:333]: *collection

**Comment by stephenhi** [comment]
We need to agree that removing 'label' is indeed the correct outcome - it is not used anywhere but does add some context?! Currently we are removing it.

**Review by jonrobins — APPROVED** [review]

File: reference/Upload-API.yaml (hunk 1 above)
Comment (stephenhi) [reply reference/Upload-API.yaml:396]: Yep not sure 7:14 is ISO 8061 so yes must be HH:mm

File: reference/Upload-API.yaml (hunk 2 above)
Comment (stephenhi) [reply reference/Upload-API.yaml:333]: Wil change.

**Review by rodek — APPROVED** [review]

**Pull request merged**